src/libs/ajax.js

```diff
@@ -238,8 +238,6 @@ export const DAR = {
     darInfo.country = rawDar.country;
     darInfo.status = rawDar.status;
     darInfo.restrictions = rawDar.restrictions;
-    darInfo.hasAdminComment = researcher.rationale != null;
```

Comment:
This is used by ApplicationSummary.js:
```
          div({ isRendered: darInfo.hasAdminComment, className: 'row no-margin' }, [
            span({}, [
              label({ className: 'control-label no-padding' }, ['Comments: ']),
              span({ id: 'lbl_adminComment', className: 'response-label', style: { 'paddingLeft': '5px' } },
                [darInfo.adminComment])
            ])
          ]),
```
I suggest that we remove that div altogether.